### PR TITLE
Improved explanation of facts-service-url parameter

### PR DESCRIPTION
**Pull request by abravosuse**
An Improved explanation of the right value of the facts-service-url parameter in the agent configuration file.

Once reviewed, this PR should be merged and published (no need to wait for new version release).

### CODE REVIEW

File: trento/xml/article_sap_trento.xml [hunk 1]
```diff
@@ -748,8 +748,15 @@ As agreed on https://confluence.suse.com/x/DAEcN on our Trento doc kick off
         </para>
         <itemizedlist>
           <listitem>
-            <para><parameter>facts-service-url</parameter>: the address of the AMQP service shared with the checks engine, where fact gathering requests are received.
-           The right syntax is <uri>amqp://rabbitmq_user:rabbitmq_user_password@<replaceable>TRENTO_SERVER_HOSTNAME</replaceable>:5672/vhost</uri>. In a &k8s; deployment, the default rabbitmq_user is trento and the default rabbitmq_user_password is trento as well. In a systemd or containerized deployment, the rabbitmq_user and the rabbitmq_user_password are set by the user during the deployment process. The corresponding installation sections above set rabbitmq_user as trento_user and rabbitmq_user_password as trento_user_password. But obviously these are just examples and should be adjusted by the customer.</para>
+            <para><parameter>facts-service-url</parameter>: the address of the AMQP RabbitMQ service used
+            for communication with the checks engine (wanda). The right value of this parameter depends on
```
Comment (dmpop) [inline trento/xml/article_sap_trento.xml:752]:
```suggestion
            for communication with the checks engine (wanda). The correct value of this parameter depends on
```

File: trento/xml/article_sap_trento.xml [hunk 2]
```diff
@@ -748,8 +748,15 @@ As agreed on https://confluence.suse.com/x/DAEcN on our Trento doc kick off
...
+            how Trento Server was deployed.</para>
+            <para> In a Kubernetes deployment, it should be amqp://trento:trento@TRENTO_SERVER_HOSTNAME:5672/
+             unless we have used helm to update the default RabbitMQ username and password (trento),
+             in which case we should use the custom value. </para>
+            <para> In a systemd or containerized deployment, it should be amqp://trento_user:trento_user_password@TRENTO_SERVER_HOSTNAME:5672/vhost
+             unless we have replaced trento_use and trento_user_password with custom values, in which case we should use them.
```
Comment (dmpop) [inline trento/xml/article_sap_trento.xml:758]:
```suggestion
            <para> In a Kubernetes deployment, it is amqp://trento:trento@TRENTO_SERVER_HOSTNAME:5672/. If the default RabbitMQ username and password (trento) were updated using helm, 
            the parameter must use the custom value. </para>
            <para> In a systemd or containerized deployment, the correct value is amqp://trento_user:trento_user_password@TRENTO_SERVER_HOSTNAME:5672/vhost. If trento_use and trento_user_password were replaced with custom values, you can use them.
```

File: trento/xml/article_sap_trento.xml [hunk 3]
```diff
@@ -748,8 +748,13 @@ As agreed on https://confluence.suse.com/x/DAEcN on our Trento doc kick off
         </para>
         <itemizedlist>
           <listitem>
-            <para><parameter>facts-service-url</parameter>: the address of the AMQP service shared with the checks engine, where fact gathering requests are received.
-           The right syntax is <uri>amqp://rabbitmq_user:rabbitmq_user_password@<replaceable>TRENTO_SERVER_HOSTNAME</replaceable>:5672/vhost</uri>. In a &k8s; deployment, the default rabbitmq_user is trento and the default rabbitmq_user_password is trento as well. In a systemd or containerized deployment, the rabbitmq_user and the rabbitmq_user_password are set by the user during the deployment process. The corresponding installation sections above set rabbitmq_user as trento_user and rabbitmq_user_password as trento_user_password. But obviously these are just examples and should be adjusted by the customer.</para>
+            <para><parameter>facts-service-url</parameter>: the address of the AMQP RabbitMQ service used
+            for communication with the checks engine (wanda). The correct value of this parameter depends on
+            how Trento Server was deployed.</para>
```
Comment (tomschr) [inline trento/xml/article_sap_trento.xml:753]:
```suggestion
            how &t.server; was deployed.</para>
```

File: trento/xml/article_sap_trento.xml [hunk 4]
```diff
@@ -748,8 +748,13 @@ As agreed on https://confluence.suse.com/x/DAEcN on our Trento doc kick off
...
+            <para> In a Kubernetes deployment, it is amqp://trento:trento@TRENTO_SERVER_HOSTNAME:5672/. If the default RabbitMQ username and password (trento) were updated using helm,
```
Comment (tomschr) [inline trento/xml/article_sap_trento.xml:754]:
```suggestion
            <para> In a Kubernetes deployment, it is <uri>amqp://trento:trento@TRENTO_SERVER_HOSTNAME:5672/</uri>. If the default RabbitMQ username and password (trento) were updated using helm, 
```

File: trento/xml/article_sap_trento.xml [hunk 5]
```diff
@@ -748,8 +748,13 @@ As agreed on https://confluence.suse.com/x/DAEcN on our Trento doc kick off
...
+            <para><parameter>facts-service-url</parameter>: the address of the AMQP RabbitMQ service used
+            for communication with the checks engine (wanda). The correct value of this parameter depends on
+            how Trento Server was deployed.</para>
+            <para> In a Kubernetes deployment, it is amqp://trento:trento@TRENTO_SERVER_HOSTNAME:5672/. If the default RabbitMQ username and password (trento) were updated using helm, 
+            the parameter must use the custom value. </para>
+            <para> In a systemd or containerized deployment, the correct value is amqp://trento_user:trento_user_password@TRENTO_SERVER_HOSTNAME:5672/vhost. If trento_use and trento_user_password were replaced with custom values, you can use them.
```
Comment (tomschr) [inline trento/xml/article_sap_trento.xml:756]: Use upper case for placeholders:

```suggestion
            <para> In a systemd or containerized deployment, the correct value is <uri>amqp://TRENTO_USER:TRENTO_USER_PASSWORD@TRENTO_SERVER_HOSTNAME:5672/vhost</uri>. If <replaceable>TRENTO_USER</replaceable> and <replaceable>TRENTO_USER_PASSWORD</replaceable> were replaced with custom values, you can use them.
```

**Review by tomschr — CHANGES_REQUESTED** [review]
@abravosuse Minor markup suggestions

**Review by abravosuse — COMMENTED** [review]
LGTM @tomschr . Thank you! Ready to go!

**Review by tomschr — APPROVED** [review]

**Pull request merged**